source/includes/_tipser-elements.md

```diff
@@ -60,7 +60,7 @@ Complete working example could look like:
     <div id="tipser_store"></div>
     <script src="https://cdn.tipser.com/tipser-script/latest.js"></script>
     <script>
-      tipser.elements('5075d7715c3d090a90585e87')
+      tipser.elements('59e86b79b8f3f60a94ecd26a')
```

Comment:
Why different POS id is used here?

Reply:
The new one is Tipser POS Id, the other is of some existing client. I didn't think it very elegant to expose it in our docs even as an example.